6.py

```diff
@@ -0,0 +1,6 @@
+a=2
+b=5
+c=6
+d=f"{str(a)}{str(b)}{str(c)}"
```

Comment:
f-strings allow skipping transformation of `a, b, c` variables and just write it like `f{a}{b}{c}`